voxblox_ros/include/voxblox_ros/mesh_vis.h

```diff
@@ -83,8 +97,8 @@ inline void heightColorFromVertex(const Point& vertex,
 }
 
 inline void fillMarkerWithMesh(const MeshLayer::ConstPtr& mesh_layer,
```

Comment:
Obviously we talked about this already... I have changed this to a constant ref in my branch.

But in general I think we should come to some standard on the use of smart pointers. Obviously they are great, but I think that we should avoid as must as possible passing them around unless they are held for non trivial time. Do you know of a standard on their use?